### ISSUE TITLE
add hint to check device permissions for minicom

### ISSUE THREAD

**Issue by niftyhacking** [johnwalicki/RaspPi-Pico-Examples-Fedora#1]
On default Fedora 34 permissions look like:
    $ ls -l /dev/ttyACM0
    crw-rw----. 1 root dialout 166, 0 Aug  9 11:33 /dev/ttyACM0

Add yourself to group dialout and 
    $ newgrp dialout

There is no informative error if permissions are incorrect for this command.
    $ minicom -b 115200 -o -D /dev/ttyACM0

The size of this "issue" is pico. There are other solutions. Running as root is at the end of the list but might be diagnostic.

Thank you.
Tom

**Comment by johnwalicki** [comment]
Good point, I often use `miniterm` and `screen` but agree that the dialout group permissions are necessary.  I'll add some words about it.
